types/lodash/lodash-tests.ts

```diff
@@ -5331,8 +5342,14 @@ fp.now(); // $ExpectType number
     _({ a: value }).get("a", defaultValue); // $ExpectType string | boolean
     _({ a: undefined }).get("a", defaultValue); // $ExpectType boolean
     _({ a: [{ b: { c: 3 } }] }).get("a[0].b.c", 0);  // $ExpectType number
+    _(objectWithOptionalField).get("a", defaultValue);  // $ExpectType boolean
+    _({}).get("a", defaultValue);  // $ExpectType boolean
+    _({}).get("a" as string, defaultValue);  // $ExpectType boolean
+    _(objectWithOptionalField).get("a", undefined);  // $ExpectType boolean | undefined
 
     _.chain("abc").get(1); // $ExpectType StringChain
+    _.chain({ a: false }).get("a"); // $ExpectType PrimitiveChain<false> | PrimitiveChain<true>
```

Comment:
Is it possible to definition return type only `PrimitiveChain<false>`?

Reply:
`{ a: false }` is inferred as `{ a: boolean }` in this case.
In case of stricter `_.chain({ a: false } as const).get("a");`, it would return only `PrimitiveChain<false>`

Reply:
Got it! thank you for your answer😘